api/rds.proto

```diff
@@ -101,6 +101,24 @@ message RouteMatch {
   repeated HeaderMatcher headers = 6;
 }
 
+message CorsPolicy {
+  // Specifies the origins that will be allowed to do CORS requests.
+  // Can be "*" to match any origin or a list of space separated origins.
+  string allow_origin = 1;
+  // Specifies the content for the access-control-allow-methods header.
+  string allow_methods = 2;
+  // Specifies the content for the access-control-allow-headers header.
+  string allow_headers = 3;
+  // Specifies the content for the access-control-expose-headers header.
+  string expose_headers = 4;
+  // Specifies the content for the access-control-max-age header.
+  string max_age = 5;
+  // Specifies whether the resource allows credentials.
+  google.protobuf.BoolValue allow_credentials = 6;
+  // Specifies if CORS is enabled.
+  google.protobuf.BoolValue enabled = 7;
```

Comment:
What is the purpose of this field? When would someone specify a CorsPolicy but have this be false?

Reply:
This only makes sense on a route. So you can disable CORS for a specific route. I have to amend the code for that. Do you think we should split the CorsPolicy into two, one for the route and one for the virtual host? Alternatively we could use the same message but ignore the field for the virtual host and add a note in the documentation?

Reply:
I see. I'm fine with having the same message used in both places with better field documentation. @htuch @alyssawilk any preference?